modules/op-about-tasks.adoc

```diff
@@ -36,9 +36,27 @@ spec: <4>
           oc apply -f $(params.manifest_dir)
           echo -----------------------------------
 ----
-<1> The Task API version, `v1beta1`.
+<1> The task API version, `v1beta1`.
 <2> The type of Kubernetes object, `Task`.
-<3> The unique name of this Task.
-<4> The list of parameters and Steps in the Task and the workspace used by the Task.
+<3> The unique name of this task.
+<4> The list of parameters and steps in the task and the workspace used by the task.
 
-This Task starts the pod and runs a container inside that pod using the specified image to run the specified commands.
+This task starts the pod and runs a container inside that pod using the specified image to run the specified commands.
+
+[NOTE]
+====
+Starting with {pipelines-shortname} {pipelines-ver}, the steps in a task does not have the following defaults:
```

Comment:
I suggest we avoid using {pipelines-ver} here and actually mention the version. When these docs get carried forward to the next version and we change the version attribute for that version, it will change the version here as well. So if you intend to say starting version 1.6 here, it will change to starting 1.7 for the next version. I suppose we do not want that to happen in this case. Please correct me if I have not understood this correctly.

Reply:
the steps in a task does not/the steps in a task do not - subject-verb agreement
IMHO, it is easy for a busy user to skip the do not/does not part from the stem sentence from the above. How about:
...the  following defaults from the `step` YAML file have been removed. ?
More importantly I find that as a user my attention is drawn straight to the bullet points, so can we say this: 
* The `HOME` environment variable does not default to the `/tekton/home` directory
* The `workingDir` field does not default to the `/workspace` directory
WDYT?